### ISSUE TITLE
Simplify Hello World test suite

### ISSUE THREAD

**Issue by raphalvessa** [exercism/dart#57]
The tests for the Hello World exercise in Dart test a `helloWorld.hello()` function that can be called with a parameter `name`. The description of the exercise doesn't mention any function parameters, and looking at the same exercise in other languages, none of them test for function parameters. I suggest sticking to the exercise description, and removing the tests that expect a `name`. After all, the Hello World should be the simplest, most basic exercise to introduce the language to new coders.

**Comment by Stargator** [comment]
I see your point @raphalvessa.

I looked at the [specification for hello world](https://github.com/exercism/problem-specifications/blob/master/exercises/hello-world/canonical-data.json) and it doesn't have a case for parameters.

And I have looked at a handful of other tracks and could find a parameter in their hello world exercise. So I agree with you.

But I'll rope in our new maintainers @fwip @kabiir to see if they want to chime in.
